sql/_35_fig_cake/grant_revoke_redefine/cases/cbrd_25627_04.sql

```diff
@@ -0,0 +1,58 @@
+/* Verfified the CBRD-25627
+As-is: if changed the owner of objects then it is not updated on db_auth
+To-be: If changed the owner of objects then revoke all grantors
+*/
+
+
+
+create user u1;
+create user u2;
+create user u3;
+
+evaluate 'case 4: changed owner for table (with grant alter privilege)';
+
+evaluate 'create u1.tbl, u1.tbl2 table';
+create table u1.tbl (col1 int);
+create table u1.tbl2 (col1 int);
+
+evaluate 'connect to u1 & u1.tbl, u1.tbl2 grant alter to u2';
+call login('u1','') on class db_user;
+GRANT ALTER ON u1.tbl TO u2;
+GRANT ALTER ON u1.tbl2 TO u2 WITH GRANT OPTION;
+
+evaluate 'connect to u2 & u1.tbl2 grant alter to u3';
+call login('u2','') on class db_user;
+GRANT ALTER ON u1.tbl2 TO u3 WITH GRANT OPTION;
+
+evaluate 'connect to dba';
+call login('dba','') on class db_user;
+select grantor_name, grantee_name, owner_name, object_type, object_name, auth_type from db_auth where grantee_name != 'PUBLIC' order by grantor_name;
```

Comment:
```suggestion
select grantor_name, grantee_name, owner_name, object_type, object_name, auth_type, is_grantable from db_auth where grantee_name != 'PUBLIC' order by grantor_name, object_type;
```
WITH GRANT OPTION 이 사용되었으므로 is_grantable 추가하는 것이 좋겠고, 정렬 조건에 object_type 추가하는 것이 좋겠습니다.

Reply:
반영했습니다.
리뷰 감사합니다.